apps/dashboard/src/app/account/account-kbs/account-kbs.component.ts

```diff
@@ -125,7 +115,6 @@ export class AccountKbsComponent implements OnInit, OnDestroy {
           this.setLoading(true);
           return new WritableKnowledgeBox(this.sdk.nuclia, this.account!.slug, kb).modify({ state });
         }),
-        switchMap(() => this.updateKbs()),
```

Comment:
It seems that when a kb is published/unpublished, the button text doesn't change